### ISSUE TITLE
Basic binary classification case

### ISSUE THREAD

**Issue by jwitos** [fabiankueppers/calibration-framework#17]
Hi, I'm having problems understanding what's the proper use of the library for a very simple binary classifier. I have a 1-D array of binary labels {0, 1} and a 1-D array of model predictions with probability values _p_ in range (0, 1). Those values reflect the probability of a positive class.

Plugging those values into e.g. the reliability diagram, I got the following plot:
![image](https://user-images.githubusercontent.com/948674/126897599-28fc79d9-45fd-41aa-bce4-306735e4c651.png)
Confidence histogram makes sense to me, as most samples are negative and classifier correctly assigns a low probability. But I'm not sure how to interpret the reliability diagram -- what do the dark red bars suggest here? Also, ECE I received is very high (>0.8).

I tried to reverse the probabilities for negative samples, i.e. if a label is 0, then the probability is _(1-p)_. This gives a more justifiable plot:
![image](https://user-images.githubusercontent.com/948674/126897674-8b2075ad-18b5-4dd1-a2e9-2466b8c60426.png)

Could you confirm that for negative samples the probability should reflect probability of a negative class, not the positive class, even in a binary classification case?

Also, would you mind clarifying whether the predictions (confidence estimates) used as inputs should be values before or after sigmoid/softmax? Temperature scaling paper says that scaling should be done on logits (pre-sigmoid) values, but reading the docs I couldn't find any information if different methods should use logits versus probabilities after sigmoid or softmax. Thank you!
